### PR TITLE
Fixed the broken "connect 2 router/subnet faq"

### PR DESCRIPTION
**Pull request by gbuerger**
Added a new step, changed picture in the right order and also added a translation for the added step (including 4 new pictures)

### CODE REVIEW

File: english/docs/documentation/networking.md [hunk 1]
```diff
@@ -29,32 +29,44 @@ Sign in at the  [Dashboard](https://dashboard.cloud.syseleven.net)
...
+To add a static route:
+* Click on the left side bar, then "Network" --> "Router".
+* Click on the first router you want to establish a connection with.
+* Click on "Static Route"
+* Click "Add Static Route" and enter the ip-range from the target network and as "Nexthop" the same ip-adress as in Step Two
 
-#### Step Three: Creating the Host Route
+Repeat the process with the other router
+
+![Interface Übersicht](../img/static-route.png)
```
Comment (stefreak) [inline english/docs/documentation/networking.md]: Image description is still german

File: english/docs/documentation/networking.md [hunk 2]
```diff
@@ -29,32 +29,44 @@ Sign in at the  [Dashboard](https://dashboard.cloud.syseleven.net)
 #### Step Two: Create Interface on Router
 
 To create a new interface on our router:
-* Click on the left side bar, then "network" --> "router".
+* Click on the left side bar, then "Network" --> "Router".
 * Click on the first router you want to establish a connection with.
-* In the new window, under "interfaces", click on "add interfaces".
-* Click on the router we want to connect to and add an IP under "IP" from the related subnet. 
+* In the new window, under "Interfaces", click on "Add Interfaces".
+* Click on the router we want to connect to and add an IP under "IP" out of the ip-range from target router .
```
Comment (stefreak) [inline english/docs/documentation/networking.md]: space before period / full stop.

File: german/docs/documentation/networking.md [hunk 3]
```diff
@@ -6,44 +6,62 @@
 
 SysEleven Stacks Networking Service basiert auf dem OpenStack Neutron Projekt.
 
-Sowohl via unserer öffentlichen OpenStack API, als auch durch das SysEleven Stack Dashboard können sie ihr virtualisierter Netzwerk verwalten.
+Damit kannst du Netowrk-Connectivity-as-a-Service für andere Dienste im SysEleven Stack nutzen wie zum Beispiel den Compute Service.
+Außerdem stellt es Usern eine API zur Verfügung um Netzwerke zu definieren und um bestehende anzupassen.
+
+Du hast zwei Möglichkeiten dein Netzwerk zu verwalten. Du kannst dafür die öffentliche OpenStack API benutzen, genau so wie das [Dashboard](https://dashboard.cloud.syseleven.net).
```
Comment (stefreak) [inline german/docs/documentation/networking.md]: Do we have a guideline wether we use "Du" or "Sie" (or completely avoid both of them)?

Comment (gbuerger) [inline german/docs/documentation/networking.md]: I'm not totally sure about it, but as fare as i know, we use "Du" on our homepage.
So i would say we should use "Du" also for our Documentation

Comment (stefreak) [inline german/docs/documentation/networking.md]: sounds plausible to me. @LarsFronius you have an opinion on that?

**Pull request merged**